### PR TITLE
Update README.md example of log params 

### PR DESCRIPTION
**Pull request by shtirlic**
Shorter example of log params

### CODE REVIEW

**Comment by pxlpnk** [comment]
Thank you, I am not sure if this change helps the understanding of the code though.
What was your intention behind this?

**Comment by t3hk0d3** [comment]
+1. This is much better.

**Comment by benlovell** [comment]
It certainly golfs down a little, but it doesn't add any clarity.

**Comment by shtirlic** [comment]
Hi Andreas,  the first thing I did after enabling lograge was searching for "params" in Readme file and I expected to find clear and small one-liner to cutpasteforget into production env file and I think this is 99.9% of the cases for the example. For people who are really want to understand better the process there is no problem to read simple one-liner.

**Comment by benlovell** [comment]
I wonder if `controller` and `action` are common enough that we should provide API for these, almost as a special case?

**Comment by pxlpnk** [comment]
@shtirlic I see your point. But maybe putting this into an api as @benlovell mentioned would make more sense and could also be clearer!

@benlovell how would you see that API?

**Comment by shtirlic** [comment]
Something like
`config.lograge.include_params = true `
would be great.

**Comment by benlovell** [comment]
@shtirlic Exactly, something like your example would work...

However, the more I think about this, the more of a philosophical question it becomes, given the opinionated nature of this gem.

@pxlpnk Opinions on making this behaviour a first-class citizen?

**Comment by pxlpnk** [comment]
sorry for the delay.
@shtirlic I like that actually. And I think we should have both ways possible @benlovell.
Although lograge is opinionated this is actually something that I use myself and I have the feeling others do it too.

@shtirlic Would you mind crafting a PR for this?

**Comment by benlovell** [comment]
@shtirlic any interest in crafting a PR for this feature? Otherwise I'm sure either @pxlpnk or myself will pick this up.

File: README.md [hunk 1]
```diff
@@ -247,11 +247,7 @@ add it in manually using `custom_options`:
 YourApp::Application.configure do
   config.lograge.enabled = true
   config.lograge.custom_options = lambda do |event|
-    params = event.payload[:params].reject do |k|
-      ['controller', 'action'].include? k
-    end
-
-    { "params" => params }
+    { params: event.payload[:params].reject { |k| %w(controller action).include? k } }
```
Comment (morgoth) [inline README.md]: even shorter:
`{params: event.payload[:params].except("action", "controller")}`

Comment (aripollak) [inline README.md]: This is what we've been using: 
```ruby
config.lograge.custom_options = -> (event) do
  { params: event.payload[:params].except(*%w(action controller format id)) }
end

**Comment by shtirlic** [comment]
There are many ways to do it, we need to benchmark all of them for speed

File: README.md (hunk 1 above)
Comment (pxlpnk) [inline README.md]: Yeah same here, seems like a very common pattern for the params.

**Comment by pxlpnk** [comment]
@shtirlic I am happy to assist you in this if you need support.

**Pull request closed without merging**